distro/runtime/concierge/smarthome.xargs

```diff
@@ -0,0 +1,236 @@
+# Concierge start profile for Eclipse SmartHome
+
+# Version configuration
+-Dconcierge.version=5.0.0
+-Djetty.version=9.3.10
+-Desh.version=0.9.0.b1
+
+# uncomment to clean storage first
+-Dorg.osgi.framework.storage.clean=onFirstInit
+
+# use our own profile
+-Dorg.eclipse.concierge.profile=eclipse-smarthome
+
+# configure Concierge debug configuration
+# -Dorg.eclipse.concierge.log.enabled=true
+# -Dorg.eclipse.concierge.log.quiet=false
+# -Dorg.eclipse.concierge.log.buffersize=1000
+
+# 4=DEBUG, 3=INFO, 2=WARN, 1=ERROR, 0=NULL
+# -Dorg.eclipse.concierge.log.level=3
+# -Dorg.eclipse.concierge.debug=true
+# -Dorg.eclipse.concierge.debug.bundles=true
+# -Dorg.eclipse.concierge.debug.packages=true
+# -Dorg.eclipse.concierge.debug.services=true
+# -Dorg.eclipse.concierge.debug.classloading=true
+
+# Basic directory - Used for either standard script or when running in IDE
+-Dbase.dir=.
+
+# bundle directories
+-Dsystem.dir=${base.dir}/system
+-Dconcierge.dir=${base.dir}/bundles
+-Djetty.dir=${system.dir}/org.eclipse.jetty
+-Desh.dir=${system.dir}/org.eclipse.smarthome
+
+# OSGi properties needed for export java internal packages
+-Dorg.osgi.framework.bootdelegation=\
+   com.sun.org.apache.xerces.internal.jaxp,\
+   sun.misc,\
+   sun.reflect
+
+
+-Dorg.osgi.framework.system.packages.extra=\
+   javax.activation,\
+   javax.crypto,\
+   javax.crypto.spec,\
+   javax.inject,\
+   javax.imageio,\
+   javax.imageio.spi,\ 
+   javax.imageio.stream,\ 
+   javax.naming,\
+   javax.naming.directory,\
+   javax.naming.ldap,\
+   javax.naming.spi,\
+   javax.net,\
+   javax.net.ssl,\
+   javax.management,\
+   javax.management.modelmbean,\
+   javax.management.remote,\
+   javax.script,\
+   javax.security.auth,\
+   javax.security.auth.callback,\
+   javax.security.auth.login,\
+   javax.security.auth.spi,\ 
+   javax.security.auth.x500,\
+   javax.security.cert,\
+   javax.sql,\
+   javax.xml,\
+   javax.xml.bind,\
+   javax.xml.bind.annotation,\
+   javax.xml.bind.annotation.adapters,\
+   javax.xml.datatype,\
+   javax.xml.namespace,\
+   javax.xml.parsers,\
+   javax.xml.stream,\
+   javax.xml.transform,\
+   javax.xml.transform.dom,\
+   javax.xml.transform.sax,\
+   javax.xml.transform.stream,\
+   javax.xml.validation,\
+   javax.xml.xpath,\
+   org.ietf.jgss,\
+   org.w3c.dom,\
+   org.w3c.dom.bootstrap,\
+   org.w3c.dom.ls,\
+   org.xml.sax,\
+   org.xml.sax.helpers,\
+   sun.misc
+
+
+# OSGi shell. Toggle between concierge shell | Apache Felix GoGo
+-istart ${concierge.dir}/org.eclipse.concierge.shell-${concierge.version}*.jar
+# -istart ${system.dir}/org.apache.felix.gogo.runtime-0.*.jar
+# -istart ${system.dir}/org.apache.felix.gogo.command-0.*.jar
+# -istart ${system.dir}/org.apache.felix.gogo.shell-0.*.jar
+ 
+# Concierge extensions, services (fragment, do NOT start)
+#-install ${concierge.dir}/org.eclipse.concierge.extension.permission-${concierge.version}*.jar
```

Comment:
I would remove that line. If needed only include the OSGi permission classes as service bundle, and not as framework extension